### PR TITLE
Britech-299 Change service endpoints to just the service name at port…

### PR DESCRIPTION
**Pull request by dwybourn**
… 8080.

### CODE REVIEW

**Comment by jebbinSCL** [comment]
:shipit:

File: ApiGateway/ProfilesRepository.fs [hunk 1]
```diff
@@ -8,7 +8,7 @@ open Serilog
 open Dtos
 open Models
 
-let profilesUri = "http://api.bris.tech/profiles/"
+let profilesUri = "http://sessions:8008/profiles/"
```
Comment (pallenSL) [inline ApiGateway/ProfilesRepository.fs]: Do we mean 8008 ?

**Comment by pallenSL** [comment]
Also I don't understand why one of these is in App.config but not all of them.

**Comment by pallenSL** [comment]
or how these service names get resolved, for that matter, but I presume that is dealt with somewhere else.

File: ApiGateway/ProfilesRepository.fs (hunk 1 above)
Comment (dwybourn) [inline ApiGateway/ProfilesRepository.fs]: I meant 8080, well spotted.

**Comment by jebbinSCL** [comment]
Ah, that is true. They do need to be moved into the config.

**Comment by dwybourn** [comment]
I did think it was strange that the endpoints weren’t in the config, the auth service endpoint is.  I guess that can be refactored at a later date.

**Comment by dwybourn** [comment]
Docker swarm and consul provide a fast refresh DNS server that is used for service discovery. I’ll be explaining this more in the knowledge share tomorrow.

**Comment by pallenSL** [comment]
Ok, that'll do for now.
:ship:

**Pull request merged**